test/connectivity/scripts/connectivityTest.sh

```diff
@@ -343,6 +343,10 @@ function run_connectivity_test() {
     print_highlighted_message "Run connectivity test with -d '$device_id' started at $test_start_time"
 
     SECONDS=0
+    PARENT_HOSTNAME=$(printEnv E2E_PARENT_HOSTNAME)
```

Comment:
Is `printEnv` a valid bash command?

Reply:
I think so, but wrong case.  should be "printenv".